### ISSUE TITLE
Json

### ISSUE THREAD

**Issue by shugo111** [Rushyanth111/Semster-Stats#1]
Need a api which contains the json data as follows student wise
-Name 
-Usn 
-Section 
-subject codes
-internal marks 
-external marks
-total
-Pass/Fail/absent
-total marks 
-overall marks
-overall pass/fail

**Comment by Rushyanth111** [comment]
I'm going to assume that you require the following:
```jsonc
{
    "Scheme":2015, //Integer
    "Batch":2016, //Integer
    "Semester":6, //Integer
}
```

An Example Response (If given to the above.) Will be:

```jsonc
[{
    "Name":"SomeName",
    "USN":"1CX16CS017",
    "Section":"", //Currently Unavailable.
    "Marks":[ //An Array of Marks, Per Code.
        {
            "Code":"15CS61",
            "Internal":18, //Integer 
            "External":56, //Integer
            "Total":64, //Integer
            "Result":"Pass",
            "Class":"FCD",
        },
        //Similar Objects as above.
    ],
    "Overall":{
        "Total":560,
        "Result":"Pass",
    } //Only One Object.
}
// More Objects Like the one Above, Based on your request.
]
```

**Comment by Rushyanth111** [comment]
Is this Acceptable? I can change it if needed.

**Comment by shugo111** [comment]
This is more than enough for now

**Comment by Rushyanth111** [comment]
Fixed with #3 

You are free to Clone the repository or call to the API at https://semdata.rxav.pw

Note that only 16CS Sem 6, 7 and 17CS Sem 5 is available.

**Comment by Rushyanth111** [comment]
Any Other Request Will Return a 500, or a Bad response.
